delfin/tests/unit/drivers/fujitsu/eternus/test_eternus_stor.py

```diff
@@ -14,6 +14,8 @@
 import sys
 from unittest import TestCase, mock
 
+from flask import ctx
```

Comment:
If ctx is not used, set this parameter to None. Otherwise, use from delfin.context import RequestContext instead.
![image](https://user-images.githubusercontent.com/59107077/153791802-8c8354c3-4dc2-45ea-9e5f-11a839bc75fe.png)

Reply:
Has been modified